src/views/docs/en/reference/cli/hydrate.md

```diff
@@ -1,77 +1,40 @@
 ---
 title: hydrate
 category: CLI
-description: Quickly install and update dependencies for all functions in the src directory.
+description: Install and update dependencies for all functions in a project.
 ---
 
-## Overview
+Ensure that all functions managed by Architect have their dependencies installed. Functions containing all its required dependencies are considered to be "hydrated" - thus the name!
 
-[`@architect/hydrate`](Architect env) ensures that all functions managed by architect have their dependencies installed. Functions containing all its required dependencies are considered to be 'hydrated' - thus the name!
+Importantly, `arc hydrate` will also copy shared code (from `src/shared` and `src/views`) into all functions.
```

Comment:
`arc hydrate` only copies `src/views` into functions that are GET routes right?

Reply:
Good point! I will update. ty

Reply:
Yes, and also:
```
@http
get /
get /foo
get /lol
post /bar
patch /fiz

@views
get /foo
post /bar
patch /fiz
```

In the above scenario, of the 5 routes listed, only the 3 under `@views` get `src/views`.